oxidation/libparsec/crates/core_fs/src/error.rs

```diff
@@ -0,0 +1,66 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use thiserror::Error;
+use uuid::Uuid;
+
+use parsec_api_crypto::CryptoError;
+use parsec_api_types::FileDescriptor;
+
+#[derive(Error, Debug)]
+pub enum FSError {
```

Comment:
Why the pascal case for the error message ?

Reply:
Is there any reason for another case ?

Reply:
I prefer phrases instead of word like:

```diff
- ConnectionError
+ connection error
```

Reply:
I like this type of error: `Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }`
Where the PascalCase is the kind of error

Reply:
You can retrieve fastly your error in that case

Reply:
#2438 I opened an issue to format Error message